editor/import/editor_scene_importer_gltf.cpp

```diff
@@ -1808,11 +1805,16 @@ void EditorSceneImporterGLTF::_generate_bone(GLTFState &state, int p_node, Vecto
 		const GLTFNode *gltf_bone_node = state.nodes[state.skins[skin].bones[n->joints[i].bone].node];
 		const String bone_name = gltf_bone_node->name;
 		const int parent = gltf_bone_node->parent;
+		if (parent == -1) {
+			continue;
+		}
 		const int parent_index = s->find_bone(state.nodes[parent]->name);
-
 		const int bone_index = s->find_bone(bone_name);
+		ERR_EXPLAIN("GLTF can't find bone parent: " + state.nodes[parent]->name);
+		ERR_CONTINUE(parent_index == -1);
+		ERR_EXPLAIN("GLTF can't find bone: " + bone_name);
+		ERR_CONTINUE(bone_index == -1);
```

Comment:
These two should now be changed to `ERR_CONTINUE_MSG` instead of using `ERR_EXPLAIN`, otherwise should be fine to merge.

Reply:
Changing.